### ISSUE TITLE
Feature: ignore certain folders/paths/globs

### ISSUE THREAD

**Issue by bgotink** [jenkinsci/violation-comments-to-stash-plugin#72]
I use this plugin to get eslint warnings in pull requests, but in PRs that contain a lot of changed files (1,000+) that eslint doesn't comment on (in our case: zipfiles) this violation command takes a very long time doing over 1,000 unneeded HTTP requests. This takes easily over 15 minutes if Jenkins / the network / BitBucket is already under a higher than normal load.

It would help a lot if we could tell the command that changes to zipfiles—alternatively, the `.yarn/cache` folder—won't trigger any warnings so it can skip those.

**Comment by tomasbjerre** [comment]
Sounds like you have `.yarn/cache` committed in version control?
You are talking about all the HTTP requests taking a lot of time right?

**Comment by bgotink** [comment]
> Sounds like you have `.yarn/cache` committed in version control?

Indeed. We're using yarn 2, which—by default—places its cache inside the repository. It's a tradeoff between repo size and speed of `yarn install` / dependency on the npm registry.

> You are talking about all the HTTP requests taking a lot of time right?

Yes, the 1,000+ HTTP requests that the plugin makes for those zipfiles take a long time.

**Comment by tomasbjerre** [comment]
There is such an option now. Released as `1.126`.

```groovy
   ignorePaths: [
       [path: 'node_modules'],
       [path: '.yarn']
   ],
```

**Comment by bgotink** [comment]
Great, thanks!
